src/lib.rs

```diff
@@ -13,6 +13,7 @@ pub mod components;
 pub mod conversions;
 pub mod gameplay_systems;
 pub mod load;
+pub mod auto_create_config;
 
```

Comment:
Should we also add something like:

```suggestion

pub use qevy_derive;
```

So that the `QevyEntity` derive macro can be accessed from outside `qevy`? (The example doesn't have this problem since it can import `qevy_derive` directly)